templates/frontend/pages/search.tpl

```diff
@@ -18,76 +18,74 @@
  *}
 {include file="frontend/components/header.tpl" pageTitle="common.search"}
 
-<div class="container page-search">
-	<div class="page-header">
-		<h1>
-			{if $query}
-				{translate key="plugins.themes.healthSciences.search.resultsFor" query=$query|escape}
-			{elseif $authors}
-				{translate key="plugins.themes.healthSciences.search.resultsFor" query=$authors|escape}
-			{else}
-				{translate key="common.search"}
-			{/if}
-		</h1>
-	</div>
+<div class="container">
+	<h1>
+		{if $query}
+			{translate key="plugins.themes.healthSciences.search.resultsFor" query=$query|escape}
+		{elseif $authors}
+			{translate key="plugins.themes.healthSciences.search.resultsFor" query=$authors|escape}
+		{else}
+			{translate key="common.search"}
+		{/if}
+	</h1>
 	<div class="row justify-content-lg-center">
-		<div class="col-lg-8 search-col-results">
-			<div class="search-results">
+		<div class="col-lg-8">
+			<div>
 
 				{* No results found *}
 				{if $results->wasEmpty()}
 					{if $error}
-						<div class="alert alert-danger" role="alert">{$error|escape}</div>
+						<div role="alert">{$error|escape}</div>
 					{else}
-						<div class="alert alert-info" role="alert">{translate key="search.noResults"}</div>
+						<div role="alert">{translate key="search.noResults"}</div>
 					{/if}
 
 				{* Results pagination *}
 				{else}
 					{iterate from=results item=result}
 						{include file="frontend/objects/article_summary.tpl" article=$result.publishedArticle journal=$result.journal showDatePublished=true hideGalleys=true}
 					{/iterate}
-					<div class="pagination">
+					<div>
```

Comment:
Here and in other places .pagination class that is dropped was used to make general rules for all pagination thought the site. Don't you think it worth to leave it?

Reply:
Let’s put it back and make it into a component! It’s definitely used a bit everywhere in different contexts

Reply:
Created a separate issue - #15